src/multiportgraph.rs

```diff
@@ -872,4 +873,20 @@ pub(crate) mod test {
 
         Ok(())
     }
+
+    #[test]
```

Comment:
Link here for reference
```suggestion
    #[test]
    /// <https://github.com/CQCL/portgraph/pull/191>
```